### DISCUSSION TITLE
`page-navigation` for blog listings

### DISCUSSION THREAD

**Discussion by mccarthy-m-g** [quarto-dev/quarto-cli#3775] [Q&A] [answered]
Is it possible to get `page-navigation` for blog listings? (e.g., like the ones you see at the bottom of [this page](https://quarto.org/docs/authoring/markdown-basics.html#other-blocks) in the Quarto docs)

My expectation as a user is that if `page-navigation: true` then the posts in a listing would have page navigation, based on [this](https://quarto.org/docs/websites/website-navigation.html#page-navigation) documentation:

> If you have a website with several pages in a section or subsection, it is often convenient to offer the user the ability to navigate to the next page (or previous page) at the bottom of the page that they’ve just finished reading. 

But this isn't the case. For example, the [Quarto blog's posts](https://quarto.org/docs/blog/posts/2022-10-25-shinylive-extension/) don't have page navigation even though `_quarto.yml` contains `page-navigation: true`. 

If this is possible, what needs to be done to make it work?

___

In relation to the quoted documentation above, it seems like "section" or "subsection" refer to a YAML `section` parameter, rather than a casual use of the word. If this is the case, it would be nice to see the documentation reflect that to prevent misunderstandings.

https://github.com/quarto-dev/quarto-web/blob/f5e07a442eaa4528ab2bff5079ab98065fdd02d2/_quarto.yml#L107-L127

**Comment by dragonstyle** [comment, marked as answer]
You are correct that page navigation will navigate between pages within a sidebar (whether arranged in sections or not). It currently cannot navigate between posts in a blog or listing (as it discovers the page and their order from their position within a sidebar definition). To improve this, we would need to determine whether the page is participating in a listing and use that listing ordering to discover the next/previous page. This won't be trivial but is possible. Feel free to open an issue to track making the improvement!

**Reply by mccarthy-m-g** [reply]
Thanks! Opened an issue :)

**Reply by gadenbuie** [reply]
Here's a link to the issue for anyone else who would like to upvote the feature request: https://github.com/quarto-dev/quarto-cli/issues/3795
